charts/minecraft-bedrock/values.yaml

```diff
@@ -22,6 +22,21 @@ tolerations: []
 
 affinity: {}
 
+podSecurityContext:
+  runAsUser: 1000
+  runAsGroup: 3000
+  runAsNonRoot: true
+  fsGroup: 2000
```

Comment:
@uhthomas I had missed this before: why did you choose group ID 3000 to run and group ID for the filesystem? It seems to be causing issues like #164

Reply:
This is standard as recommended by the official Kubernetes documentation.

https://kubernetes.io/docs/tasks/configure-pod-container/security-context/

`uid=1000 gid=3000 groups=2000`

The idea is that there is as little permission as possible.

Reply:
Thanks for following up.